ru/passwords.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    "password" => "Пароль должен быть не менее шести символов и совпадать с подтверждением.",
+    "password" => "Пароль должен содержать не менее шести символов и совпадать с полем подтверждения пароля.",
```

Comment:
@EvgenyKovalev I'm so sorry, but I have translated auth.php, for example) That's okay, no problem, I don't insist to use my version)